ConcernsCaseWork/ConcernsCaseWork.Service/Trusts/TrustService.cs

```diff
@@ -12,16 +14,21 @@ public sealed class TrustService : TramsAbstractService, ITrustService
 	{
 		private readonly ILogger<TrustService> _logger;
 		private readonly IFakeTrustService _fakeTrustService;
+		IFeatureManager _featureManager;
+
+		private const string EndpointV3 = "v3";
 
 		public TrustService(
 			IHttpClientFactory clientFactory, 
 			ILogger<TrustService> logger, 
 			ICorrelationContext correlationContext, 
 			IClientUserInfoService userInfoService,
-			IFakeTrustService fakeTrustService) : base(clientFactory, logger, correlationContext, userInfoService)
+			IFakeTrustService fakeTrustService,
+			IFeatureManager featureManager) : base(clientFactory, logger, correlationContext, userInfoService)
```

Comment:
Are we ok with adding the Feature Management packages across our application? Abstracting it out seems a little overkill but as things group it might be easier to maintain. What do you think?

Reply:
Short discussion on this and agreed the simplicity of the library means we are happy to add the package across the application